src/server/command/debugCommand.ts

```diff
@@ -0,0 +1,109 @@
+import { AnyChannel, Client, Message } from 'discord.js';
+
+async function getCommand(
+  client: Client,
+  message: Message,
+  prefix: string,
+  command: string
+) {
+  if (message.author.bot || !message.guild) return;
+
+  const str = message.content;
+  const match = str.startsWith(command);
+  if (!match) return;
+
+  const commandArgs = str.slice(prefix.length).trim().split(' ');
+  const argsMessageId = commandArgs[1];
+  if (!argsMessageId) {
+    await message.reply({
+      content: `**メッセージIDが指定されていません。**\n Usage: \`${prefix}debug <メッセージID>\``
```

Comment:
これで `prefix` 引数は不要になります
```suggestion
      content: `**メッセージIDが指定されていません。**\n Usage: \`${command} <メッセージID>\``
```